### PR TITLE
accelerate, acme.sh-dns, ant: add pt_BR translation

### PR DESCRIPTION
**Pull request by victorhtl**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page(s) follow the [style guide](/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.pt_BR/common/accelerate.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# accelerate
+
+> Uma biblioteca que habilita o mesmo código PyTorch a rodar em qualquer configuração distribuída.
+> Mais informações: <https://huggingface.co/docs/accelerate/index>.
+
+- Printa informações do ambiente:
```
Comment (renie) [inline pages.pt_BR/common/accelerate.md:6]:
```suggestion
- Mostra informações do ambiente:
```

File: pages.pt_BR/common/accelerate.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+`accelerate env`
+
+- Cria um arquivo de configuração de forma interativa:
+
+`accelerate config`
+
+- Printa o custo de memória de GPU estimado para rodar um Modelo de Face aumentado com diferentes tipos de dados:
```
Comment (renie) [inline pages.pt_BR/common/accelerate.md:14]:
```suggestion
- Mostra o custo de memória de GPU estimado para rodar um Modelo de Face aumentado com diferentes tipos de dados:
```

File: pages.pt_BR/common/accelerate.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+`accelerate estimate-memory {{name/model}}`
```
Comment (renie) [inline pages.pt_BR/common/accelerate.md:16]:
```suggestion
`accelerate estimate-memory {{nome/modelo}}`
```

File: pages.pt_BR/common/accelerate.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+- Testa um arquivo Accelerate de configuração:
+
+`accelerate test --config_file {{path/to/config.yaml}}`
```
Comment (renie) [inline pages.pt_BR/common/accelerate.md:20]:
```suggestion
`accelerate test --config_file {{caminho/para/config.yaml}}`
```

File: pages.pt_BR/common/accelerate.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+- Roda um modelo na CPU com Accelerate:
+
+`accelerate launch {{path/to/script.py}} {{--cpu}}`
```
Comment (renie) [inline pages.pt_BR/common/accelerate.md:24]:
```suggestion
`accelerate launch {{caminho/para/script.py}} {{--cpu}}`
```

File: pages.pt_BR/common/accelerate.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+
+- Roda um modelo em multi-GPU com Accelerate, com 2 máquinas:
+
+`accelerate launch {{path/to/script.py}} --multi_gpu --num_machines 2`
```
Comment (renie) [inline pages.pt_BR/common/accelerate.md:28]:
```suggestion
`accelerate launch {{caminho/para/script.py}} --multi_gpu --num_machines 2`
```

File: pages.pt_BR/common/acme.sh-dns.md [hunk 7]
```diff
@@ -0,0 +1,24 @@
+# acme.sh --dns
+
+> Usa um desafio DNS-01 para emitir um certificado TLS.
+> Mais informações: <https://github.com/acmesh-official/acme.sh/wiki>.
+
+- Emite um certificado usando um modo de DNS API automático:
+
+`acme.sh --issue --dns {{gnd_gd}} --domain {{example.com}}`
+
+- Emite um certificado wildcard (denotado por um asterisco (*)) usando um modo DNS API automático:
```
Comment (renie) [inline pages.pt_BR/common/acme.sh-dns.md:10]:
```suggestion
- Emite um certificado wildcard (denotado por um asterísco (*)) usando um modo DNS API automático:
```

File: pages.pt_BR/common/acme.sh-dns.md [hunk 8]
```diff
@@ -0,0 +1,24 @@
...
+
+`acme.sh --issue --dns {{dns_namesilo}} --domain {{example.com}} --domain {{*.example.com}}`
+
+- Emite um certificado usando um modo apelido de DNS:
+
+`acme.sh --issue --dns {{dns_cf}} --domain {{example.com}} --challenge-alias {{alias-for-example-validation.com}}`
```
Comment (renie) [inline pages.pt_BR/common/acme.sh-dns.md:16]:
```suggestion
`acme.sh --issue --dns {{dns_cf}} --domain {{example.com}} --challenge-alias {{apelido-para-exemplo-de-validacao.com}}`
```

File: pages.pt_BR/common/acme.sh-dns.md [hunk 9]
```diff
@@ -0,0 +1,24 @@
...
+
+- Emite um certificado enquanto desabilita a pesquisa automática de DNS da Cloudflare/google depois que o registro DNS for adicionado, especificando um tempo de espera personalizado em segundos:
```
Comment (renie) [inline pages.pt_BR/common/acme.sh-dns.md:18]:
```suggestion
- Emite um certificado enquanto desabilita a pesquisa automática de DNS da Cloudflare/Google depois que o registro DNS for adicionado, especificando um tempo de espera personalizado em segundos:
```

File: pages.pt_BR/common/ant.md [hunk 10]
```diff
@@ -0,0 +1,24 @@
+# ant
+
+> Apache Ant: compile e administre projetos baseados em Java.
```
Comment (renie) [inline pages.pt_BR/common/ant.md:3]:
```suggestion
> Apache Ant: compila e administra projetos baseados em Java.
```

File: pages.pt_BR/common/ant.md [hunk 11]
```diff
@@ -0,0 +1,24 @@
...
+> Mais informações: <https://ant.apache.org>.
+
+- Compila um projeto com o arquivo padrão de build `build.xml`:
+
+`ant`
+
+- Compila um projeto utilizando um arquivo de build diferente do `build.xml`:
+
+`ant -f {{buildfile.xml}}`
```
Comment (renie) [inline pages.pt_BR/common/ant.md:12]:
```suggestion
`ant -f {{arquivo_de_build.xml}}`
```

File: pages.pt_BR/common/ant.md [hunk 12]
```diff
@@ -0,0 +1,24 @@
...
+
+- Printa informações sobre possíveis alvos para este projeto:
```
Comment (renie) [inline pages.pt_BR/common/ant.md:14]:
```suggestion
- Mostra informações sobre possíveis alvos para este projeto:
```

File: pages.pt_BR/common/ant.md [hunk 13]
```diff
@@ -0,0 +1,24 @@
...
+
+`ant -p`
+
+- Printa informações de debug:
```
Comment (renie) [inline pages.pt_BR/common/ant.md:18]:
```suggestion
- Mostra informações de debug:
```

**Review by renie — CHANGES_REQUESTED** [review]
Thanks for you contribution @victorhtl !

**Review by renie — APPROVED** [review]
All good now! Thanks for the contribution

**Comment by victorhtl** [comment]
@sebastiaanspeck it just coincidence haha I'm another person and, actually, this is my first contribution to an open source project

**Comment by renie** [comment]
I also had the same though of @sebastiaanspeck . Interesting coincidence!

**Pull request merged**